### PR TITLE
Mplayer menu mod

### PR DESCRIPTION
**Pull request by ggm-at-apnic**
### All Submissions:

- [*] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew/blob/master/.github/CONTRIBUTING.md) document?
- [*] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew/pulls) for the same update/change?

I've modifed the .rb file, to explicitly include the requested else --disable-x11 behaviour, so that the default and normal build has no x11 dependencies. But, if you enable the menu feature, you get x11.

This means you can have an OSD with x11, or the default no OSD no x11

### CODE REVIEW

File: Library/Formula/mplayer.rb [hunk 1]
```diff
@@ -45,11 +48,20 @@ def install
       --host-cc=#{ENV.cc}
       --disable-cdparanoia
       --prefix=#{prefix}
-      --disable-x11
     ]
 
     args << "--enable-caca" if build.with? "libcaca"
 
+    # to get OSD to work on El Capitan we seen to need x11
```
Comment (ilovezfs) [inline Library/Formula/mplayer.rb]: seem not seen

**Comment by mikemcquaid** [comment]
There's more context as to why this was removed in https://github.com/Homebrew/homebrew/pull/46749.

**Comment by mikemcquaid** [comment]
I'd probably rather we didn't add new graphical options that need X11 as it's not been shipped with OS X for a long time.

**Comment by ggm-at-apnic** [comment]
ok. I understand your preferences. I'll keep my own strand of logic alive
to do OSD until they tear the code out in mplayer central. I don't want a
GTK+ GUI, and I like having keyboard accellerators and an overlay inside
the view pane, but I get that other people don't and you want a simpler
non-X life in OSX.

cheers

-George

**Comment by DomT4** [comment]
Closing per Mike's comments then. Thank you for the submission nonetheless :heart:

**Pull request closed without merging**